### PR TITLE
fix: exponential fog

### LINKED ISSUE #699: Fix fog configuration
### **Issue Description:**
The fog is in a state of flux atm.
We should enforce EXP and remove the others, and fix the scene shader so it's capable of EXP.

### PR DESCRIPTION
**Pull request by GBirch33**
Fixes #699

Test: load up the scene and switch between quality settings verifying fog in the distance is visible and doesn't cause any rendering issues.

### CODE REVIEW

File: Explorer/Assets/Scripts/Localization/Test/Italian (it).asset
Comment (olavra) [inline Explorer/Assets/Scripts/Localization/Test/Italian (it).asset:1]: Italian localization is back and prevents the client from starting

File: Explorer/Assets/Scripts/Localization/Test/Spanish (es).asset
Comment (olavra) [inline Explorer/Assets/Scripts/Localization/Test/Spanish (es).asset:1]: Nobody expects spanish localization. It breaks the initialization of the game

File: Explorer/Assets/DCL/StylizedSkybox/Scripts/SkyboxController.cs
Comment (olavra) [inline Explorer/Assets/DCL/StylizedSkybox/Scripts/SkyboxController.cs:1]: Will this prevent the skybox from changing the values of fogDensity if we decide to control it over time?

Comment (GBirch33) [reply Explorer/Assets/DCL/StylizedSkybox/Scripts/SkyboxController.cs:1]: No it'll be able to be modified by Skybox in the future, this should solve some possible conflicts between different systems.

**Comment by GBirch33** [comment]
> Fog is still not applied to the Scene Shader.
> 
> image

When the assets get rebuilt then it should work. Current version doesn't have Exp fog compiled in the shader.

**Comment by m3taphysics** [comment]
*PENDING* LOD Generation

**Comment by GBirch33** [comment]
@anicalbano this is now testable

**Comment by m3taphysics** [comment]
image

**Review by m3taphysics — APPROVED** [review]

**Pull request merged**